# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1337365109 25200
#      Fri May 18 11:18:29 2012 -0700
# Node ID f19d29868346e63145c7849c59b2d896d3f5649e
# Parent  8f792cff711ecf176f426d7dfc1b11e2eb017bc0
build: Distribute functions in the polynomial/private directory.

* polynomial/module.mk: Add private fcn files to list in
polynomial_FCN_FILES variable.

diff --git a/scripts/polynomial/module.mk b/scripts/polynomial/module.mk
--- a/scripts/polynomial/module.mk
+++ b/scripts/polynomial/module.mk
@@ -1,12 +1,12 @@
 FCN_FILE_DIRS += polynomial
 
 polynomial_PRIVATE_FCN_FILES = \
-	polynomial/private/__splinefit__.m
+  polynomial/private/__splinefit__.m
 
 polynomial_FCN_FILES = \
   polynomial/compan.m \
   polynomial/conv.m \
   polynomial/deconv.m \
   polynomial/mkpp.m \
   polynomial/mpoles.m \
   polynomial/pchip.m \
@@ -23,15 +23,16 @@ polynomial_FCN_FILES = \
   polynomial/ppval.m \
   polynomial/ppder.m \
   polynomial/ppint.m \
   polynomial/ppjumps.m \
   polynomial/residue.m \
   polynomial/roots.m \
   polynomial/spline.m \
   polynomial/splinefit.m \
-  polynomial/unmkpp.m
+  polynomial/unmkpp.m \
+  $(polynomial_PRIVATE_FCN_FILES)
 
 FCN_FILES += $(polynomial_FCN_FILES)
 
 PKG_ADD_FILES += polynomial/PKG_ADD
 
 DIRSTAMP_FILES += polynomial/$(octave_dirstamp)
